site/en/tutorials/images/_pix2pix.ipynb

```diff
@@ -608,6 +626,15 @@
         "* The discriminator receives the input_image and the generated image as the first input. The second input is the input_image and the target_image.\n",
         "* Next, we calculate the generator and the discriminator loss.\n",
         "* Then, we calculate the gradients of loss with respect to both the generator and the discriminator variables(inputs) and apply those to the optimizer.\n",
+        "* This entire procedure is shown in the images below.\n",
+        "\n",
+        "![alt text](https://raw.githubusercontent.com/yashk2810/yashk2810.github.io/master/dis.jpg)\n",
```

Comment:
Replace "alt text".

Yes, add an ./images directory to put these instead of your personal account (similar to the text_generation notebook in [sequences/](https://github.com/tensorflow/docs/tree/master/site/en/tutorials/sequences)).

Reply:
Done.